docs/src/dataDisplay/table/index.jsx

```diff
@@ -1,210 +1,443 @@
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleVerticalAlign').default}
+                >
+                    <TableSampleVerticalAlign />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Table Header Cell onClick Event Handler
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        A table header cell can handle an onClick event.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleOnClick').default}
+                >
+                    <TableSampleOnClick />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Responsive Width
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        A table header cell and table ceel can specify a width
+                        for a specific device.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleResponsiveWidth').default}
+                >
+                    <TableSampleResponsiveWidth />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Drag and Drop Row Reordering
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        A table&rsquo;s rows can be drag and drop enabled to allow reordering.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!./tableSampleDragAndDropRowReordering').default}
+                >
+                    <TableSampleDragAndDropRowReordering />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="children"
+                        variant="h2"
+                    >
+                        Sticky Columns
+                    </Heading>
+
+                    {/* <Typography
+                        variant="body1"
+                    >
+                        // Need Description.
+                    </Typography> */}
```

Comment:
Indeed it does need a description!

Please use the following:

> A table can have one or more columns defined as 'sticky' so that they stay fixed while the user horizontally scrolls to see the remaining columns.

Reply:
Fixed.

Reply:
Looks good; resolving.